_dev/src/ts/api/SegmentApi.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License version 3.0
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
+ */
+import { AnalyticsBrowser } from '@segment/analytics-next';
+import { maskSensitiveInfoInUrl } from '../utils/urlUtils';
+
+class Analytics {
+  analytics: AnalyticsBrowser;
+
+  public constructor() {
+    this.analytics = new AnalyticsBrowser();
+    this.analytics.load({ writeKey: 'RM87m03McDSL4Fvm3GJ3piBPbAL3Fa2i' });
```

Comment:
```suggestion
    this.analytics = AnalyticsBrowser.load(
      { writeKey: 'RM87m03McDSL4Fvm3GJ3piBPbAL3Fa2i' },
      {disableClientPersistence: true},
    );
```

This will avoid conflicts with other instances of Segment that stores identifcation data in the browser storage (see https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/cookie-validity-update/).